server/src/api/profile-clan/profile-clan-api.ts

```diff
@@ -0,0 +1,24 @@
+import { Router } from 'express';
+import { ProfileClanApiPath } from '../../common/enum/api/profile-clan-api-path';
+import { ProfileClanService } from '../../services';
+
+export const initProfileClan = (appRouter: typeof Router, services: { profileClan: ProfileClanService }) => {
+	const { profileClan: profileClanService } = services;
+
+	const router = appRouter();
+
+	router.put(ProfileClanApiPath.TO_ONE_BY_ID, async (req, res, next) => {
+		profileClanService
+			.updateRole(req.params.id,req.body.role)
+			.then((data)=>res.send(data))
+			.catch(next);
+	});
```

Comment:
Only admin of own clan can give to other members the admin role. Make it by middleware / service